### PR TITLE
📖 Update drawings for example scenario 1

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR updates the drawing for example scenario 1, to align them better with reality and the outline drawing.

## Related issue(s)

Fixes #

### CODE REVIEW

File: docs/content/en/docs/Coding Milestones/PoC2023q1/Edge-PoC-2023q1-Scenario-1-stage-4.svg
Comment (waltforme) [inline docs/content/en/docs/Coding Milestones/PoC2023q1/Edge-PoC-2023q1-Scenario-1-stage-4.svg:1]: For the two arrows from mailbox workspaces to edge clusters, we may want to make their lower-right arrow heads consistently point to the boundaries of the edge clusters, or consistently point to the boundaries of the syncer namespaces.

**Review by waltforme — COMMENTED** [review]
A tiny detail is that, blocks for all controllers are transparent except that 'Status summarizer' is opaque. I tried to download the svgs and insert them to some slices, maybe opaque blocks looks better.

File: docs/content/en/docs/Coding Milestones/PoC2023q1/Edge-PoC-2023q1-Scenario-1-stage-4.svg
Comment (MikeSpreitzer) [reply docs/content/en/docs/Coding Milestones/PoC2023q1/Edge-PoC-2023q1-Scenario-1-stage-4.svg:1]: Sorry, editing goof.  Made consistent in the latest revision.

**Comment by MikeSpreitzer** [comment]
@waltforme : the latest revision makes all the controller boxes opaque.

**Pull request merged**